lmms_eval/models/chat/async_openai.py

```
@@ -184,11 +184,17 @@ async def maybe_forward_with_tool(self, request: Instance, idx: int):
                 for call in message.tool_calls:
                     eval_logger.debug(f"Calling {call.function.name}...")
                     result = await self.mcp_client.run_tool(call.function.name, eval(call.function.arguments))
```

Ruff (0.12.2)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)